src/chaosbag/BlessCurseManager.ttslua

```diff
@@ -71,8 +67,7 @@ end
 
 function initializeState()
     -- count tokens in the bag
-    local chaosbag = getChaosBag()
-    if chaosbag == nil then return end
+    local chaosbag = Global.call("findChaosBag")
```

Comment:
implemented usage of central findChaosBag function instead of having different variations on every object